### PR TITLE
Enable using envar for password

### PR DESCRIPTION
**Pull request by liiight**
This will look for `PYTEST_JIRA_PASSWORD` envvar and use it if exist (take precedence over config password).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -192,6 +192,12 @@ Usage
      # run_test_case = True (default value for 'run' parameter)
      # connection_error_strategy [strict|skip|ignore] Choose how to handle connection errors
 
+   You can set the password field by setting the PYTEST_JIRA_PASSWORD environment variable:
+
+    .. code:: sh
+
+      set PYTEST_JIRA_PASSWORD="FOO"
```
Comment (lukas-bednar) [inline README.rst]: didn't you mean `export` instead of `set` ?

Comment (liiight) [reply README.rst]: yeah, oops :)

**Comment by lukas-bednar** [comment]
It looks great!
Test is failing because of #96 ... I will fix that so we can pass ci for this pr.

**Comment by liiight** [comment]
Cool. I don't know why my git history is fucked up like that, I swear I rebased. I'll reclone after this is merged...

**Comment by lukas-bednar** [comment]
@liiight could you please rebase your change ?
Travis should pass now!

File: pytest_jira.py [hunk 2]
```diff
@@ -143,8 +144,8 @@ def is_affected(self, issue_id):
         else return False
         """
         return (
-            self._affected_version(issue_id) and
-            self._affected_components(issue_id)
+                self._affected_version(issue_id) and
```
Comment (lukas-bednar) [inline pytest_jira.py]: this is unnecessary indent.

Comment (liiight) [reply pytest_jira.py]: pycharm like it this way apparantly... fixed

**Comment by lukas-bednar** [comment]
Thank you !

**Comment by liiight** [comment]
Sure thing. Thanks for this plugin 👍 
Note that I pinged you about #94

**Pull request merged**